2013-05-12  Aurelien Jarno  <aurelien@aurel32.net>

	* locales/bo_CN (LC_NAME): set name_fmt to "???".
	* locales/bo_IN (LC_NAME): Likewise.

--- a/localedata/locales/bo_CN
+++ b/localedata/locales/bo_CN
@@ -145,7 +145,7 @@ END LC_MEASUREMENT
 LC_NAME
 % FIXME
 
-name_fmt  ""
+name_fmt  "???"
 % name_gen	"FIXME"
 % name_miss	"FIXME"
 % name_mr	"FIXME"
--- a/localedata/locales/bo_IN
+++ b/localedata/locales/bo_IN
@@ -70,7 +70,7 @@ END LC_MEASUREMENT
 
 LC_NAME
 % FIXME
-name_fmt	""
+name_fmt	"???"
 % name_gen	"FIXME"
 % name_miss	"FIXME"
 % name_mr	"FIXME"
